frontend/src/app/Settings/ManageOrganization.tsx

```diff
@@ -8,16 +8,9 @@ import Alert from "../commonComponents/Alert";
 import Select from "../commonComponents/Select";
 import { OrganizationTypeEnum } from "../signUp/Organization/utils";
 
-import { EditableOrganization } from "./ManageOrganizationContainer";
-
-type OrganizationFormData = {
-  name: string;
-  type: OrganizationType;
-};
-
 interface ManageOrganizationProps {
-  organization: EditableOrganization;
-  onSave: (organization: EditableOrganization) => Promise<void>;
+  organization: OrganizationForm;
```

Comment:
This organization prop should map to the `Organization` type defined in `../../generated/graphql` as is basically receiving the endpoint response.